### PR TITLE
refactor: Shape up WynntilsAccountManager

### PR DESCRIPTION
**Pull request by magicus**
Make it behave more like the other managers, and expose sensible methods to users

### CODE REVIEW

File: common/src/main/java/com/wynntils/commands/TokenCommand.java [hunk 1]
```diff
@@ -25,39 +24,34 @@ public LiteralArgumentBuilder<CommandSourceStack> getBaseCommandBuilder() {
...
+        }
+
+        String token = WynntilsAccountManager.getToken();
+
+        MutableComponent text = new TextComponent("Wynntils Token ").withStyle(ChatFormatting.AQUA);
+        MutableComponent response = new TextComponent(token)
+                .withStyle(Style.EMPTY
+                        .withHoverEvent(new HoverEvent(
+                                HoverEvent.Action.SHOW_TEXT,
+                                new TextComponent("Click me to register an" + " account.")))
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/commands/TokenCommand.java]:
```suggestion
                                new TextComponent("Click me to register an account.")))
```

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/commands/TokenCommand.java]: (just noticed this, I know you didn't touch it)

File: common/src/main/java/com/wynntils/sockets/model/HadesModel.java [hunk 2]
```diff
@@ -56,8 +56,8 @@ public static void disable() {
     }
 
     private static void tryCreateConnection() {
-        if (WynntilsAccountManager.getAccount().isEmpty()) {
-            WynntilsMod.error("Cannot connect to HadesServer when WebManager does not have account.");
+        if (!WynntilsAccountManager.isLoggedIn()) {
+            WynntilsMod.error("Cannot connect to HadesServer when not logged in on Athena.");
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/sockets/model/HadesModel.java]:
```suggestion
            WynntilsMod.error("Cannot connect to HadesServer when your account is not logged in on Athena.");
```

File: common/src/main/java/com/wynntils/commands/TokenCommand.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/commands/TokenCommand.java]: Heh. I actually fixed that in my original webapi rewrite branch. But when I extracted and ported those changes over to a new main-based branch, I missed it. Thanks for noticing!

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**